### DISCUSSION TITLE
duplication

### DISCUSSION THREAD

**Discussion by priyang12** [faker-js/faker#1601] [Q&A]
I am trying to use faker in a seeder function where I need a unique email but `faker.internet.email()` gives the same email each time (i am running in node.js).

**Comment by ST-DDT** [comment]
@priyang12 Can you give us a little bit more context what you are doing/what your implementation is like? Which version are you using
Is it possible that you set the seed to the exact same value each time before generating the email?
You can also try to use [faker.helpers.unique](https://fakerjs.dev/api/helpers.html#unique)
